lte/gateway/c/oai/common/common_types.h

```diff
@@ -238,10 +238,10 @@ typedef union {
 typedef enum {
   BPS  = 0,
   KBPS = 1,
-} apn_ambr_bits_unit_t;
+} apn_ambr_bitrate_unit_t;
 
 typedef struct {
-  apn_ambr_bits_unit_t bitsunit;
+  apn_ambr_bitrate_unit_t bitrateunit;
```

Comment:
let's make this naming consistent with `br_ul` and `br_dl`, i.e., `br_unit`.

Reply:
Resolved